routes.js

```diff
@@ -1,86 +1,91 @@
-//     });
-//     res.sendStatus(200);
-// });
+// Define routes
 
+membersAPI.get('/getAllMembers', (req, res) => {
+  try {
+    const data = loadData();
+    res.status(200).send(data);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
 
-// createMyCustomApi.put('/updateExistMember', (req, res) => {
-//   const {memberId, role , firstname, lastname} = req.body
-//   if(!memberId) {
-//     return res.status(400).send('Member ID is required');
-//   }
-//   const memberIndex = data.findIndex(member => member.memberId === memberId);
-//   if (memberId === -1) {
-//     return res.status(404).send("Member not found");
-//   }
-//   const updatedMember = {
-//     memberId,
-//     role : role,
-//     firstname : firstname ,
-//     lastname : lastname
-//   };
 
-//   data[memberIndex] = updatedMember
+membersAPI.post('/addNewMember', (req, res) => {
+  try {
+    let myObject = loadData();
+    let newData = req.body; // required body from postman
 
+    const { memberId, role, firstname, lastname } = newData;
+    if (!memberId || !role || !firstname || !lastname) {
+      return res.status(400).send('All fields (memberId, role, firstname, lastname) are required');
+    }
 
-//   saveData(data)
-//   res.status(201).send('Member updated successfully');
-// });
+    myObject.push(newData); // Pushing body into data
+    saveData(myObject); // Save the updated data
+    res.sendStatus(200);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
 
 
+membersAPI.put('/updateExistMember', (req, res) => {
+  const { memberId, role, firstname, lastname } = req.body
+  if (!memberId) {
+    return res.status(400).send('Member ID is required');
+  }
+  const memberIndex = data.findIndex(member => member.memberId === memberId);
+  if (memberId === -1) {
+    return res.status(404).send("Member not found");
+  }
+  const updatedMember = {
+    memberId,
+    role: role,
+    firstname: firstname,
+    lastname: lastname
+  };
 
-// let data = require('./members_Array.json'); // swsto datapath
+  data[memberIndex] = updatedMember
 
 
-// const saveData = (data) => {
-//   fs.writeFileSync('members_Array.json', JSON.stringify(data, null, 2)); // gia na grapsoume sto json arxeio
-// };
+  saveData(data)
+  res.status(201).send('Member updated successfully');
+});
 
-// createMyCustomApi.delete(`/removeMember/:memberId`, (req, res) => {
-//     const memberId = req.params.memberId;       
-//     const initialLength = data.length; // store initial length of our data
 
+membersAPI.delete(`/removeMember/:memberId`, (req, res) => {
+  const memberId = req.params.memberId;
+  let data = loadData();
+  const initialLength = data.length; // store initial length of our data
 
-//     data = data.filter(member => member.memberId !== memberId); // filtrarisma , create new array , xwris to member pou diagrafoume
+  data = data.filter(member => member.memberId !== memberId); // filtrarisma , create new array without the deleted one.
```

Comment:
Curious idea, it will work!
Wondering if something like this will be self-explanatory and not require an additional comment? https://www.w3schools.com/howto/howto_js_remove_property_object.asp

Reply:
Whoops..! its Greek-lish, never mind me!